client/src/stories/spriteAnimation.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef } from "react";
+import { playerImage } from "./assets/spriteSheet";
+
+/*
+one image: 180 x 240 = srcWidth x srcHeight
+rows & cols = 4
+*/
+
+function drawPlayerSprite(player, context) {
+  const { width, height, left, top, direction } = player;
+
+  const srcWidth = 180;
+  const srcHeight = 240;
+  const startCol = 0;
+  let startRow;
+
+  switch (direction) {
+    case "FRONT":
+      startRow = 0;
+      break;
+    case "LEFT":
+      startRow = 1;
+      break;
+    case "RIGHT":
+      startRow = 2;
+      break;
+    case "JUMPING":
+      startRow = 3;
+      break;
+    default:
+      startRow = 0;
+      break;
+  }
```

Comment:
As an alternative:

```suggestion
  const directionRows = {
    FRONT: 0,
    Left: 1,
    Right: 2,
    JUMPING: 3
  };
  const startRow = directionRows[direction] || directionRows.FRONT;
```

Reply:
> Awesome sprites! Did you draw it by yourself? 
Yes. This weekend 😄 

> 
oh nice. I will definitely take this one. Is there a reason Left and Right are not in capitals, or could I write them in capitals as well?

Reply:
You can write them as capitals as well :)